### DISCUSSION TITLE
Message tab doesnt update on next page

### DISCUSSION THREAD

**Discussion by chg-dbyron** [provectus/kafka-ui#4022] [General]
When clicking the next page button at the bottom of the messages tab, the page appears to change but all of the offsets stay the same. I am wondering if anyone else has seen this?

Currently using version 7.0 via helm

**Comment by chg-dbyron** [comment]
Also, I am able to work around this by doing any of the following:

1. change the “seek type” to timestamp, and filtering by date (mixed results)
2. enter the next offset after the last offset displayed on the page in the offset field, and it brings up a new page starting from that offset on.
3. modify the limit parameter in the url

**Comment by chg-dbyron** [comment]
same issue that is shown [here](https://github.com/provectus/kafka-ui/issues/3129).

**Comment by Haarolean** [comment]
Hi, that's a known issue (#3129), and will be fixed after implementing #3504 on frontend.
